python/hsfs/storage_connector.py

```diff
@@ -110,77 +110,167 @@ def connector_type(self):
     @property
     def access_key(self):
         """Access key."""
-        return self._access_key
+        if self._storage_connector_type == "S3":
```

Comment:
The user is not able to set these properties in any way right? they are always coming in capital letters from the backend?
Just want to make sure we don't need to lower/upper case these things.

Reply:
Yes, but it's easy enough to add an upper case. Let me do it .

Reply:
Done